src/basis.jl

```diff
@@ -91,7 +162,23 @@ end
 #  pair basis 
 
 
-"""TODO add documentation"""
+"""
+`pair_basis_params(; kwargs...)` : returns a dictionary containing the 
+complete set of parameters required to construct an pair basis (`PolyPairBasis`). 
+All parameters are passed as keyword argument. 
+
+### Parameters
+* `species` : single species or list of species (mandatory)
+* `maxdeg` : maximum degree, positive real number (note the precise notion of degree 
+is specified by further parameters) (mandatory)
+* `r0 = 2.5` : rough estimate for nearest neighbour distance
+* `rcut = 5.0`: outer cuttoff, Å 
+* `rin = 0.0`: inner cuttoff, Å 
+* `pcut = 2`: outer cutoff parameter
```

Comment:
Should what these values mean be documented, or is the user meant to understand that from reading some paper?

Reply:
I skipped over this because I wasn't entirely sure myself. As far as I know, p=2 means potential and first derivative go to zero at the cutoff, p=0 unconstrained? What about p=1? @cortner would you mind clarifying?

Reply:
Yup, I think it's that. Added that to the docstrings.